### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by dkranke**

### CODE REVIEW

File: src/main/resources/assets/supercircuitmaker/lang/de_de.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  "container.supercircuitmaker.constant.dec_20": "-20",
+  "container.supercircuitmaker.constant.dec_1": "-1",
+  "container.supercircuitmaker.constant.inc_1": "+1",
+  "container.supercircuitmaker.constant.inc_20": "+20",
+
+  "subtitles.supercircuitmaker.place": "BOOP",
+
+  "key.supercircuitmaker": "Super Circuit Maker",
+  "key.supercircuitmaker.open_palette": "Öffne Palette",
+  "key.supercircuitmaker.component_placement_modifier": "Komponenten Platzierungsmodifikator",
```
Comment (2xsaiko) [inline src/main/resources/assets/supercircuitmaker/lang/de_de.json:46]:
```suggestion
  "key.supercircuitmaker.component_placement_modifier": "Komponenten-Platzierungsmodifikator",
```

File: src/main/resources/assets/supercircuitmaker/lang/de_de.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  "item.supercircuitmaker.note": "Booper",
+  "item.supercircuitmaker.lamp": "Miniaturlampe",
+  "item.supercircuitmaker.button": "Miniaturknopf",
+  "item.supercircuitmaker.adder": "Addierer",
+  "item.supercircuitmaker.subtractor": "Subtrahierer",
+  "item.supercircuitmaker.multiplier": "Multiplikator",
+  "item.supercircuitmaker.divider": "Teiler",
+  "item.supercircuitmaker.constant": "Konstante",
+  "item.supercircuitmaker.platform": "Support-Plattform",
+  "item.supercircuitmaker.redstone_stick": "Mit Redstone beschichteter Stock",
```
Comment (2xsaiko) [inline src/main/resources/assets/supercircuitmaker/lang/de_de.json:22]:
```suggestion
  "item.supercircuitmaker.redstone_stick": "Redstone-beschichteter Stock",
```
Maybe? Shorter and imo less awkward. But it doesn't work with the RGB stick so maybe not, to keep it consistent

File: src/main/resources/assets/supercircuitmaker/lang/de_de.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{
+  "itemGroup.supercircuitmaker": "Super Circuit Maker",
+
+  "block.supercircuitmaker.circuit": "Schaltkreis",
+  "block.supercircuitmaker.inspector": "Inspektor",
+  "item.supercircuitmaker.screwdriver": "Schraubendreher",
+  "item.supercircuitmaker.palette": "Palette und Pinsel",
+  "item.supercircuitmaker.tiny_redstone": "Kleiner Haufen von Redstone Staub",
+  "item.supercircuitmaker.tiny_rgb_redstone": "Kleiner Haufen von buntem Redstone Staub",
```
Comment (2xsaiko) [inline src/main/resources/assets/supercircuitmaker/lang/de_de.json:9]:
```suggestion
  "item.supercircuitmaker.tiny_redstone": "Kleiner Haufen von Redstone-Staub",
  "item.supercircuitmaker.tiny_rgb_redstone": "Kleiner Haufen von buntem Redstone-Staub",
```

Maybe also *Häufchen* to make it shorter? Or even
```suggestion
  "item.supercircuitmaker.tiny_redstone": "Häufchen Redstone-Staub",
  "item.supercircuitmaker.tiny_rgb_redstone": "Häufchen bunter Redstone-Staub",
```
which should also be correct, I believe?

File: src/main/resources/assets/supercircuitmaker/lang/de_de.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+  "subtitles.supercircuitmaker.place": "BOOP",
+
+  "key.supercircuitmaker": "Super Circuit Maker",
+  "key.supercircuitmaker.open_palette": "Öffne Palette",
+  "key.supercircuitmaker.component_placement_modifier": "Komponenten Platzierungsmodifikator",
+
+  "msg.supercircuitmaker.inspector.learned": "Gelernt!",
+  "msg.supercircuitmaker.inspector.mode_state": "Ausgewählter Modus: Blockzustand-Überstimmung",
+  "msg.supercircuitmaker.inspector.mode_block": "Ausgewählter Modus: Blockart-Überstimmung"
```
Comment (2xsaiko) [inline src/main/resources/assets/supercircuitmaker/lang/de_de.json:50]:
```suggestion
  "msg.supercircuitmaker.inspector.mode_state": "Ausgewählter Modus: Blockzustand vergleichen",
  "msg.supercircuitmaker.inspector.mode_block": "Ausgewählter Modus: Blockart vergleichen"
```
Or
```suggestion
  "msg.supercircuitmaker.inspector.mode_state": "Ausgewählter Modus: Blockzustand-Übereinstimmung",
  "msg.supercircuitmaker.inspector.mode_block": "Ausgewählter Modus: Blockart-Übereinstimmung"
```

File: src/main/resources/assets/supercircuitmaker/lang/de_de.json [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  "container.supercircuitmaker.constant": "Konstante: ",
+  "container.supercircuitmaker.constant.dec_20": "-20",
+  "container.supercircuitmaker.constant.dec_1": "-1",
+  "container.supercircuitmaker.constant.inc_1": "+1",
+  "container.supercircuitmaker.constant.inc_20": "+20",
+
+  "subtitles.supercircuitmaker.place": "BOOP",
+
+  "key.supercircuitmaker": "Super Circuit Maker",
+  "key.supercircuitmaker.open_palette": "Öffne Palette",
```
Comment (2xsaiko) [inline src/main/resources/assets/supercircuitmaker/lang/de_de.json:45]:
```suggestion
  "key.supercircuitmaker.open_palette": "Palette öffnen",
```

**Review by dblsaiko — CHANGES_REQUESTED** [review]

**Review by dblsaiko — APPROVED** [review]

**Pull request merged**